### PR TITLE
Add template version and template target to template vars

### PR DESCRIPTION
**Pull request by LukasJenicek**
Cross ref. https://github.com/webrpc/gen-golang/pull/72

### CODE REVIEW

File: gen/gen.go [hunk 1]
```diff
@@ -46,6 +46,8 @@ func Generate(proto *schema.WebRPCSchema, target string, config *Config) (out *G
 		WebrpcTarget     string
 		WebrpcErrors     []*schema.Error
 		Opts             map[string]interface{}
+		TmplVersion      string
+		TmplTarget       string
```
Comment (VojtechVitek) [inline gen/gen.go:50]: Can we call this
```suggestion
		TargetName      string
		TargetVersion       string
```
to match the `-target=` CLI option?

Comment (VojtechVitek) [inline gen/gen.go:50]: Alternatively, we could call it "code generator" as per README.

```suggestion
		CodeGen        string // e.g. golang@v0.14.0
		CodeGenName    string // e.g. golang
		CodeGenVersion string // e.g. v0.14.0
```

```
  -target string
    	target code generator (required)
```

Comment (LukasJenicek) [reply gen/gen.go]: I prefer this one :)

File: gen/gen.go [hunk 2]
```diff
@@ -24,6 +24,20 @@ type GenOutput struct {
 	FormatErr error
 }
 
+type TemplateVars struct {
+	*schema.WebRPCSchema
+	SchemaHash       string
+	WebrpcGenVersion string
+	WebrpcGenCommand string
+	WebrpcGenHeader  string
```
Comment (VojtechVitek) [inline gen/gen.go:32]:
```suggestion
	WebrpcHeader  string
```

File: gen/gen.go [hunk 3]
```diff
@@ -24,6 +24,20 @@ type GenOutput struct {
 	FormatErr error
 }
 
+type TemplateVars struct {
```
Comment (VojtechVitek) [inline gen/gen.go:27]: 👍 good idea to make this a type

File: gen/gen.go [hunk 4]
```diff
@@ -117,3 +130,17 @@ func getWebrpcGenCommand() string {
...
 }
+
+func webRPCGenHeader(t TemplateVars) string {
+	webrpcGenVersion := fmt.Sprintf("webrpc@%s", t.WebrpcGenVersion)
+
+	tmplVersion := t.CodeGenVersion
+	if tmplVersion == "" {
+		tmplVersion = "unknown"
+	}
+	tmpl := fmt.Sprintf("%s@%s", t.CodeGenName, tmplVersion)
```
Comment (VojtechVitek) [inline gen/gen.go:141]:
```suggestion
	codeGenVersion := fmt.Sprintf("%s@%s", t.CodeGenName, tmplVersion)
```

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Pull request merged**